setup.cfg

```diff
@@ -1,7 +1,49 @@
+[metadata]
+name = PGPy
+author_email = mgreene@securityinnovation.com
+description = Pretty Good Privacy for Python
+keywords = OpenPGP, PGP, Pretty Good Privacy, GPG, GnuPG, openpgp, pgp, gnupg, gpg, encryption, signature
+url = https://github.com/SecurityInnovation/PGPy
+#bugtrack_url = https://github.com/SecurityInnovation/PGPy/issues",
+long_description = file: README.rst
+classifiers =
+    Development Status :: 4 - Beta
+    Operating System :: POSIX :: Linux
+    Operating System :: MacOS :: MacOS X
+    Operating System :: Microsoft :: Windows
+    Intended Audience :: Developers
+    Programming Language :: Python
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Security
+    Topic :: Security :: Cryptography
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Libraries :: Python Modules
+    License :: OSI Approved :: BSD License
+
+[options]
+packages =
+    pgpy
+    pgpy.packet
+    pgpy.packet.subpackets
+install_requires = 
+    cryptography>=2.6
+    pyasn1
+    six>=1.9.0
+    singledispatch; python_version < '3.4'
+    enum34; python_version < '3.4'
+setup_requires = setuptools>=44; wheel
+# doc_requires = sphinx; sphinx-better-theme
```

Comment:
Please reformat `setup_requires` and `doc_requires` as lists like `install_requires`